src/renderer/src/components/steps/ConfigureApplication.jsx

```diff
@@ -15,8 +15,11 @@ import Title from '~/components/ui/Title'
 import '~/components/component.animation.css'
 import Forms from '@platformatic/ui-components/src/components/forms'
 import { TYPESCRIPT, JAVASCRIPT } from '~/ui-constants'
+import log from 'electron-log/renderer'
 
 const ConfigureApplication = React.forwardRef(({ onNext, onBack }, ref) => {
+  console.log = log.log
+  Object.assign(console, log.functions)
```

Comment:
Not sure it's a good idea to monkey-patch console. log.

Reply:
I taken from the https://github.com/megahertz/electron-log?tab=readme-ov-file#renderer-process

Reply:
<img width="1004" alt="Schermata 2024-02-21 alle 16 56 01" src="https://github.com/platformatic/meraki/assets/25035977/0827d0d6-a86b-440e-9098-b9d3b313175c">

Reply:
Done!